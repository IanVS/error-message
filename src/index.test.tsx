import * as React from 'react';
import { render } from '@testing-library/react';
import { RHFError } from './index';

describe('React Hook Form Error', () => {
  it('should render correctly', () => {
    const { asFragment } = render(
      <RHFError
        errors={{ test: { type: 'test' } }}
        as={<span />}
        name="test"
      />,
    );

    expect(asFragment).toMatchSnapshot();
  });
});
