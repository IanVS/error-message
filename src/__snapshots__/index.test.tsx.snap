// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`React Hook Form Error should render correctly 1`] = `[Function]`;
